objects.go

```diff
@@ -128,6 +141,39 @@ func (chain *Chain) writeOperation(verb verb, ctx *nftContext, writer io.Writer)
 	fmt.Fprintf(writer, "\n")
 }
 
+func parseChain(words []string) (*Chain, error) {
+	if len(words) < 5 {
+		return nil, fmt.Errorf("chain command should have at least 5 words")
```

Comment:
```suggestion
		return nil, fmt.Errorf("add chain command should have at least 5 words")
```

(likewise elsewhere)